tests/nightly/test_large_array.py

```diff
@@ -1199,6 +1199,250 @@ def test_full():
+    x = nd.broadcast_to(x, (LARGE_X*10, SMALL_Y//10))
+    y = nd.cos(x)
+    assert_almost_equal(y[0][0].asnumpy(), 1, atol=1e-3)
+    assert_almost_equal(y[1][1].asnumpy(), .707, atol=1e-3)
+    assert_almost_equal(y[2][2].asnumpy(), 0, atol=1e-3)
+    assert_almost_equal(y[-2][3].asnumpy(), -.707, atol=1e-3)
+    assert_almost_equal(y[-1][-1].asnumpy(), -1, atol=1e-3)
+
+
+def test_tan():
+    x = nd.array([-np.pi/4, 0, np.pi/4]).reshape(1, 3)
+    x = nd.broadcast_to(x, (LARGE_X*10, x.shape[1]))
+    y = nd.tan(x)
+    assert y[0][0] == -1
+    assert y[1][1] == 0
+    assert y[-1][-1] == 1
+
+
+def test_radians():
+    x = nd.array([0, 90, 180, 270, 360]).reshape(1, 5)
+    x = nd.broadcast_to(x, (LARGE_X*10, SMALL_Y//10))
+    y = nd.radians(x)
+    assert_almost_equal(y[0][0].asnumpy(), 0, atol=1e-3)
+    assert_almost_equal(y[1][1].asnumpy(), np.pi/2, atol=1e-3)
+    assert_almost_equal(y[2][2].asnumpy(), np.pi, atol=1e-3)
+    assert_almost_equal(y[-2][3].asnumpy(), 3*np.pi/2, atol=1e-3)
+    assert_almost_equal(y[-1][-1].asnumpy(), 2*np.pi, atol=1e-3)
+
+
+def test_degrees():
+    x = nd.array([0, np.pi/2, np.pi, 3*np.pi/2, 2*np.pi]).reshape(1, 5)
+    x = nd.broadcast_to(x, (LARGE_X*10, SMALL_Y//10))
+    y = nd.degrees(x)
+    assert_almost_equal(y[0][0].asnumpy(), 0, atol=1e-3)
+    assert_almost_equal(y[1][1].asnumpy(), 90, atol=1e-3)
+    assert_almost_equal(y[2][2].asnumpy(), 180, atol=1e-3)
+    assert_almost_equal(y[-2][3].asnumpy(), 270, atol=1e-3)
+    assert_almost_equal(y[-1][-1].asnumpy(), 360, atol=1e-3)
+
+
+def test_L2Normalization():
+    x = nd.ones((2, LARGE_X*2))
+    x[0] = 3
+    x[1] = 4
+    # Channel Mode
+    z = x.reshape(1, 2, LARGE_X*2)
+    y = nd.L2Normalization(z, mode='channel')
+    assert y[0][0][0] == 0.6
+    assert y[0][0][-1] == 0.6
+    assert y[0][1][0] == 0.8
+    assert y[0][1][-1] == 0.8
+    # Instance Mode
+    z = x.T
+    y = nd.L2Normalization(z, mode='instance')
+    assert y[0][0] == 0.6
+    assert y[0][1] == 0.8
+    assert y[-1][0] == 0.6
+    assert y[-1][1] == 0.8
+    # Spatial Mode
+    z = z.reshape(1, 200000000, 2)
+    y = nd.L2Normalization(z, mode='spatial')
+    assert y[0][0][0] == 0.6
+    assert y[0][0][1] == 0.8
+    assert y[0][-1][0] == 0.6
+    assert y[0][-1][1] == 0.8
+
+
+def test_instance_norm():
+    dtype = np.float32
+    forward_check_eps = 1E-3
+    axis = -1
+    eps = 1E-5
+    in_shape = (LARGE_X, 1, SMALL_Y)
+    ctx = mx.cpu()
+
+    def npy_instance_norm(data, gamma, beta, axis, eps=1E-5):
+        if axis < 0:
+            axis += data.ndim
+        broadcast_shape = [1 for _ in range(data.ndim)]
+        broadcast_shape[axis] = data.shape[axis]
+        mean = data.mean(axis=axis, keepdims=True).astype(dtype)
+        var = data.var(axis=axis, keepdims=True).astype(dtype)
+        std = np.sqrt(var + dtype(eps)).astype(dtype)
+        out = gamma * (data - mean) / std + \
+              beta
+        return out
```

Comment:
Documentation explaining operator: https://beta.mxnet.io/api/ndarray/_autogen/mxnet.ndarray.InstanceNorm.html
to understand how this operator functions